c_src/exml_event.c

```diff
@@ -156,22 +204,23 @@ static ERL_NIF_TERM new_parser(ErlNifEnv *env, int argc, const ERL_NIF_TERM argv
     return enif_make_tuple(env, 2, OK, parser_resource);
 };
 
+static void reset_parser_int(ErlNifEnv *env, XML_Parser parser)
```

Comment:
Do you think function name `reset_and_init_parser` is still correct and more meaningful?